netbox/utilities/forms.py

```diff
@@ -311,12 +311,17 @@ def add_filter_for(self, name, value):
 
     def add_additional_query_param(self, name, value):
         """
-        Add details for an additional query param in the form of a data-* attribute.
+        Add details for an additional query param in the form of a data-* JSON-encoded list attribute.
```

Comment:
Wouldn't it be simpler to use whitespace separation for multiple values when assigning the attribute on the form element? With the JSON approach, you end up with stuff like this:

```
data-additional-query-param-site_id="[&quot;null&quot;, 6]"
```

Reply:
Primarily to work nicely with jQuery's [`.data`](https://api.jquery.com/data/#data-html5) which attempts to deserialize the value as JSON if possible. Apart from that, I have no preference on which separator to use.